tools/nni_cmd/nnictl_utils.py

```diff
@@ -700,11 +700,11 @@ def export_trials_data(args):
                 trial_records = []
                 for record in content:
                     if not isinstance(record['value'], (float, int)):
-                        formated_record = {**record['parameter'], **record['value'], **{'id': record['id']}}
+                        formated_record = {**record['parameter'], **json.loads(record['value']), **{'id': record['id']}}
                     else:
                         formated_record = {**record['parameter'], **{'reward': record['value'], 'id': record['id']}}
                     trial_records.append(formated_record)
-                with open(args.path, 'w') as file:
+                with open(args.path, 'w', newline='') as file:
```

Comment:
why add `newline` here?

Reply:
without `newline=''`, the exporter will produce extra newline in the exported csv file

Reply:
python doc reference: https://docs.python.org/3/library/csv.html#csv.writer